opal-core-api/src/main/java/org/obiba/opal/core/domain/GenotypesMapping.java

```diff
@@ -0,0 +1,108 @@
+package org.obiba.opal.core.domain;
+
+import org.hibernate.validator.constraints.NotBlank;
+
+import javax.validation.constraints.NotNull;
+
+public class GenotypesMapping {
```

Comment:
This is not a GenotypesMapping, this is a VCF samples mapping, so rename accordingly.